packages/payload/src/utilities/parseCookies.ts

```diff
@@ -7,7 +7,15 @@ export default function parseCookies(req: Request): { [key: string]: string } {
   if (rc) {
     rc.split(';').forEach((cookie) => {
       const parts = cookie.split('=')
-      list[parts.shift().trim()] = decodeURI(parts.join('='))
+      const key = parts.shift().trim()
+      const encodedValue = parts.join('=')
+
+      try {
+        const decodedValue = decodeURI(encodedValue)
+        list[key] = decodedValue
+      } catch (e) {
+        console.error(`Error decoding cookie value for key ${key}: ${e}`)
```

Comment:
Instead of console.error'ing, we should:
```
throw new APIError(`Error decoding cookie value for key ${key}: ${e}`)
```

Reply:
@JarrodMFlesch Done.